Resources/Prototypes/_Goobstation/Changeling/Entities/Objects/Shields/shields.yml

```diff
@@ -2,7 +2,7 @@
   parent: BaseShield
   id: ChangelingShield
   name: oraganic shield
-  description: A large, fleshy shield.
+  description: A mass of tough, boney tissue. You can still see the fingers as a twisted pattern in the shield.
```

Comment:
BRITISHISM SPOTTED!!!!!!!!!!

it's "bony" in american english

Reply:
would you believe me if this was copy-pasted from tg